addOns/commonlib/src/test/java/org/zaproxy/addon/commonlib/PolicyTagsUnitTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.commonlib;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+
+class PolicyTagsUnitTest {
+
+    @Test
+    void shouldHaveAllTagsStartingWithPolicyyUnderscoreInCaps() {
+        // Given / When
+        List<PolicyTags> list =
+                Arrays.asList(PolicyTags.values()).stream()
```

Comment:
Could also use the parameterisation on the enum asserting the tag directly starts with, rather than assert on the lengths.